### PR TITLE
Added terraform config file

### LINKED ISSUE #18: Create Terraform infrastructure
Create a basic Terraform structure (https://learn.hashicorp.com/terraform/gcp/build) to deploy the following machines in the project:
* Three GCE linux instances (one for the wazuh manager, other for the Elastic stack and another for the wazuh agent).
* A VPC network where the wazuh manager and the elastic stack will be. Only the wazuh agent will be exposed to the internet (so it can receive attacks and generate security alerts).

### PR DESCRIPTION
**Pull request by karpp**
Closes #18

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    zone = "europe-west1-c"
+}
+
+resource "google_compute_network" "vpc_network" {
+  name = "terraform-network"
+}
+
+
+resource "google_compute_instance" "vm_instance_1" {
+  name         = "terraform-instance-1"
```
Comment (alejandro-bermudez) [inline terraform/main.tf:17]: terraform-instance-1 maybe is a little bit obfuscated. I would use a more descriptive name

File: terraform/main.tf [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      image = "debian-cloud/debian-9"
+    }
+  }
+
+  network_interface {
+    network = google_compute_network.vpc_network.self_link
+  }
+}
+
+resource "google_compute_route" "internet_access" {
```
Comment (alejandro-bermudez) [inline terraform/main.tf:83]: Have you test this new script with routing?

File: terraform/main.tf [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  name = "internet-access"
+  description = "Route to internet for private instances"
+  dest_range = "0.0.0.0/0"
+  network = google_compute_network.vpc_network.self_link
+  next_hop_instance = google_compute_instance.vm_instance_1.self_link
+  next_hop_instance_zone = google_compute_instance.vm_instance_1.zone
+  tags = ["server"]
+  priority = 100
+}
```
Comment (alejandro-bermudez) [inline terraform/main.tf:92]: A new line in the end of the file

File: terraform/main.tf (hunk 2 above)
Comment (karpp) [reply terraform/main.tf:83]: Yes, I've tested

Comment (alejandro-bermudez) [reply terraform/main.tf:83]: Perfect!

**Pull request merged**